Content.Shared/Pinpointer/NavMapComponent.cs

```diff
@@ -0,0 +1,32 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Timing;
+
+namespace Content.Shared.Pinpointer;
+
+/// <summary>
+/// Used to store grid poly data to be used for UIs.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed class NavMapComponent : Component
```

Comment:
Any idea what to do when station got split? Right now it will treat it as different navigation maps.